src/components/composer/composer/composer.ts

```diff
@@ -644,11 +653,16 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
    */
   private processTokenAtCursor(): void {
     let content = this.env.model.getters.getCurrentContent();
+    const token = this.env.model.getters.getTokenAtCursor();
+
+    if (token && deepEquals(this.autoCompleteState.token, token)) {
+      return;
+    }
+
```

Comment:
I would have put this before the `let content = ...` (which can also become a `const content = ...` I think, but it clearly is a detail :)